utils/sidewalk_dfu/nordic_dfu.c

```diff
@@ -63,10 +100,52 @@ static void led_action(struct k_timer *timer_id)
 
 static void exit_dfu_mode(struct k_timer *timer_id)
 {
-	LOG_ERR("DFU did not start or could not complete. Reset to exit dfu mode");
-	sys_reboot(SYS_REBOOT_COLD);
+	LOG_ERR("DFU did not start or could not complete. Exit dfu mode");
+}
+
+static void connected(struct bt_conn *conn, uint8_t err)
+{
+	struct bt_conn_info cinfo;
+	int ec;
```

Comment:
```suggestion
	int ret;
```

Reply:
This was copied from mesh sample, as I hope it will solve one of the issues I see in Jenkins, if so, I will refactor this code